graphql/generated.go

```diff
@@ -0,0 +1,2122 @@
+// Code generated by github.com/99designs/gqlgen, DO NOT EDIT.
```

Comment:
Make sure to exclude generated code from our linters

Reply:
is there a way to tell linter to ignore this file? this file will be deleted and replaced every time code gen is run; therefore ignoring the linter in this file would not do much good because it would have to be replaced every time.